hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -195,11 +195,12 @@ public Operation convert(String value) throws ParameterException {
     public String schemaProviderClassName = null;
 
     @Parameter(names = {"--transformer-class"},
-        description = "subclass of org.apache.hudi.utilities.transform.Transformer"
+        description = "A subclass or a list of subclasses of org.apache.hudi.utilities.transform.Transformer"
             + ". Allows transforming raw source Dataset to a target Dataset (conforming to target schema) before "
             + "writing. Default : Not set. E:g - org.apache.hudi.utilities.transform.SqlQueryBasedTransformer (which "
-            + "allows a SQL query templated to be passed as a transformation function)")
-    public String transformerClassName = null;
+            + "allows a SQL query templated to be passed as a transformation function). "
+            + "Pass a comma-separated list of subclass names to chain the transformations.")
+    public String transformerClassNames = null;
```

Comment:
jcommander supports parsing a list of csvs into a List<String> here already. could we switch to that

Reply:
let me check that.

Reply:
@vinothchandar jcommander converter feature seems to be the right fit. Guess we can change this variable to `List<Transformer>` and get rid of `UtilHelpers.createTransformer()`. Sounds good?

Reply:
yes :) ..